package/.coveragerc

```diff
@@ -1,2 +0,0 @@
-[run]
-plugins = Cython.Coverage
```

Comment:
Is the cython coverage still reported?

Reply:
Should do (see: https://app.codecov.io/gh/MDAnalysis/mdanalysis/compare/3736/tree/package/MDAnalysis/lib)

This file _shouldn't_ be used by coverage anyways since we force pytest to pick up the coveragerc at the top of MDA directory.